scripts/wsrep_sst_rsync.sh

```diff
@@ -402,6 +402,14 @@ then
 
     RSYNC_PID="$WSREP_SST_OPT_DATA/$MODULE.pid"
 
+    # give some time for lingering rsync from previous SST to complete
+    check_round=0
+    while check_pid $RSYNC_PID && [ $check_round -lt 10 ]
+    do
+        check_round=`expr $check_round + 1`
```

Comment:
Backticks are considered old style, arithmetic expansion by double parentheses is preferred nowadays, see for example https://tldp.org/LDP/abs/html/arithexp.html.